docs/native/src/identity/bootstrapper.rst

```diff
@@ -0,0 +1,3 @@
+
+|stub-icon| Bootstrapping OAuth2
+------------------------------------------------------
```

Comment:
Use equals signs for top-level headings. See http://wopi.readthedocs.org/en/latest/contributing/style_guide.html#heading-styles for more info.